src/popup.html

```diff
@@ -4,29 +4,46 @@
     <style>
       body {
         width: 400px;
+        font-family: Arial, Helvetica, sans-serif;
+      }
+
+      @media (prefers-color-scheme: dark) {
+        body {
+          color: #eee;
+          background: #121212;
+          color-scheme: dark;
+        }
+      }
+
+      body:not(.windows) .if-windows,
```

Comment:
Perhaps, for good measure, we could use a better class name. Maybe chatterino-is-windows so it won't clash with any class that another extension or Twitch chooses.

Reply:
Sure, note that we are the only ones on that page - it's the popup from the extension menu.